atlasdb-dbkvs-tests/build.gradle

```diff
@@ -51,6 +51,7 @@ task postgresTest(type: Test) {
 task oracleTest(type: Test) {
     maxHeapSize = "1024m"
     include '**/DbKvsOracleTestSuite.class'
+    include '**/OracleNamespaceDeleterTests.class'
     maxParallelForks 2
 }
```

Comment:
The 
```
test {
    // Tests should be included with the relevant flavoured task.
    exclude '**/*.class'
}
```
seems error prone. I'd have imagined that we'd prefer to explicitly ignore tests covered by the previous cases so that we catch any case where tests don't fit in the above categories (or mistakenly forgotten).
I think this is for another PR, but thoughts?

Reply:
Agree 👍